src/helper.py

```diff
@@ -379,3 +386,16 @@ def print_gpt(surf_data, gpt_prompt, gpt_info):
     else:
         gpt_response = gpt.openai_gpt(summary, gpt_prompt, api_key, gpt_model)
     return gpt_response
+
+
+def print_hourly_data(curr_hour):
```

Comment:
Is this function ever used? I commented it out when testing and everything (as far as I can tell) still functions properly.

Reply:
No, I don't think it is now with the newest changes based on your suggestions. Would you like me to remove it?

Reply:
Yes please, once thats done I'll merge the PR